mautrix_signal/commands/signal.py

```diff
@@ -287,3 +288,77 @@ async def raw(evt: CommandEvent) -> None:
             await evt.reply(
                 f"Got reply `{resp_type}`:\n\n```json\n{json.dumps(resp_data, indent=2)}\n```"
             )
+
+
+@command_handler(
+    needs_auth=True,
+    management_only=False,
+    help_section=SECTION_SIGNAL,
+    help_text=("Create a Signal group for the current Matrix room. "),
+)
+async def create(evt: CommandEvent) -> EventID:
+    if await po.Portal.get_by_mxid(evt.room_id):
+        return await evt.reply("This is already a portal room.")
+
+    title, about, levels, encrypted, avatar_url = await get_initial_state(
+        evt.az.intent, evt.room_id
+    )
+    #    if not title:
+    #        return await evt.reply("Please set a title before creating a Telegram chat.")
```

Comment:
```suggestion
```

Should be uncommented or removed (is signal fine with unnamed group chats? :thinking:)